pulsar-functions/worker/src/main/java/org/apache/pulsar/functions/worker/rest/api/SinkImpl.java

```diff
@@ -249,4 +255,35 @@ public SinkStatus getSinkStatus(final String tenant,
 
         return sinkStatus;
     }
+
+    @Override
+    public Object getFunctionInfo(final String tenant,
```

Comment:
We should just have another method called getSinkInfo and it should return a SinkConfig not object